### ISSUE TITLE
Player History page won't open

### ISSUE THREAD

**Issue by ReckerLang** [emmVRC/emmVRC-Public#1654]
Player History function worked fine for a while in a world. Opened it twice, but this function stopped working after approximately 1.5 hours in the same world. Clicking on the "player history" button won't bring up anything besides a clicking sound effect. Checked emmvrc log and no error was reported. This has also happened in other random worlds. Thank you in advance.

**Comment by thetrueyoshifan** [comment]
Is this issue still occurring? Can I get your latest MelonLoader log?

**Comment by ReckerLang** [comment]
Yes, you can. And I can try to upload a video of this issue somewhere if
you wish. If I may ask where is the log saved in the directory?

**Comment by ReckerLang** [comment]
Here are all the logs under MelonLoader folder

**Comment by ReckerLang** [comment]
VRChat 2022.05.22 - 07.50.14.04.mp4
<https://drive.google.com/file/d/130RcyT1BmRX5lGOB1fjG5jKwPAOr6hQ1/view?usp=drive_web>
This issue occurred in the 05-22 log. And I will attach a video to that
issue.

**Comment by thetrueyoshifan** [comment]
Please upload your video as unlisted to YouTube, as well as your log file.

**Comment by ReckerLang** [comment]
I do not think one can upload anything other than a video on YouTube

**Comment by ReckerLang** [comment]
https://youtu.be/WKoYm0mKBJI
